### ISSUE TITLE
Add 'browser' to mainFields in webpack.config.js

### ISSUE THREAD

**Issue by benbcai** [cerner/terra-dev-site#365]
# Feature Request

## Description
As a maintainer of terra-dev-site I want to add 'browser' to the [mainFields in webpack.config.js](https://github.com/cerner/terra-dev-site/blob/v6.37.0/config/webpack/webpack.config.js#L152)

So that more modern syntax (e.g. `const`) used in dependency packages are supported in older browsers (i.e. IE11).

I'll know I'm done when 'browser' is in the beginning of the list in mainFields.

## Additional Context / Screenshots
See https://github.com/cerner/terra-toolkit/issues/522.

I get the following error in IE 11 when an external dependency (i.e. [logdown](https://github.com/caiogondim/logdown.js/blob/v3.3.1/package.json#L6)) uses some modern syntax that IE 11 is unable to process.
<img width="1611" alt="Screen Shot 2021-07-14 at 2 36 31 PM" src="https://user-images.githubusercontent.com/22618655/125682848-adf819c9-a585-4090-a498-c228b9d44355.png">
